packages/pigment-css-react/src/processors/css.ts

```diff
@@ -68,7 +68,13 @@ export class CssProcessor extends BaseProcessor {
     if (callType === 'template') {
       this.handleTemplate(this.callParam, values);
     } else {
-      this.handleCall(this.callParam, values);
+      try {
+        this.handleCall(this.callParam, values);
+      } catch(e){
+        // const error = new Error((e as Error).message);
+        // error.stack = (e as Error).stack;
+        // throw error;
```

Comment:
Personally I don't see the need of handling this. The throw here is unhandled and will just crash nextjs dev-server. Any error generated by webpack is shown in the Error view in the browser. Maybe just remove?